index.html

```diff
@@ -161,6 +161,7 @@ <h5 class="display-3 text-center project-stat">100+</h5>
                             </p>
                             <h5 class="display-3 text-center project-stat">275 000+</h5>
                             <p class="text-dark text-center stats">
+                                
```

Comment:
Remove this extra line @nimeshkavindu 
```suggestion
```

Reply:
> Remove this extra line @nimeshkavindu

Should I create a new commit for this or can I amend the commit?

Reply:
Just click commit suggestion button its enough for now @nimeshkavindu

Reply:
Ok, Thanks @YoshithaRathnayake